<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MatchClass</class>
 <widget class="QMainWindow" name="MatchClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1460</width>
    <height>521</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Match</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="ImagePath">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>571</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>影像文件选择</string>
    </property>
    <widget class="QLineEdit" name="lineEditImgL">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>461</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>select the left image(不要中文路径)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="selectLeftImg">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>20</y>
       <width>91</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>SelectLeft</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditImgR">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>461</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>select the right image(不要中文路径)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="selectRightImg">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>50</y>
       <width>91</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>SelectRight</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="relateParam">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>261</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>相关系数匹配参数</string>
    </property>
    <widget class="QComboBox" name="PyramidLevel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="labelPyramid">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>金字塔层数</string>
     </property>
    </widget>
    <widget class="QComboBox" name="relateWinSize">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="currentText">
      <string>11</string>
     </property>
     <item>
      <property name="text">
       <string>11</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>9</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>13</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>15</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>17</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="labelrelateWinSize">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>60</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>相关窗口大小</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="relateThreshold">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.010000000000000</double>
     </property>
     <property name="value">
      <double>0.850000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="labelthreshold">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>90</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>相关系数阈值(请输入)</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelthreshold_2">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>120</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Harris最多提取点数</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="upmaxHarris">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>3000</number>
     </property>
     <property name="maximum">
      <number>8000</number>
     </property>
     <property name="value">
      <number>3000</number>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="LSMParam">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>90</y>
      <width>261</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>最小二乘匹配参数</string>
    </property>
    <widget class="QLabel" name="LSMlabelrelateWinSize">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>相关窗口大小</string>
     </property>
    </widget>
    <widget class="QComboBox" name="LSMrelateWinSize">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>11</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>13</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>15</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>17</string>
      </property>
     </item>
    </widget>
    <widget class="QSpinBox" name="maxIterations">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <property name="value">
      <number>25</number>
     </property>
    </widget>
    <widget class="QLabel" name="LSMmaxIterations">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>60</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>最大迭代轮次</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="LSMrelateThreshold">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.010000000000000</double>
     </property>
     <property name="value">
      <double>0.850000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="LSMlabelthreshold">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>90</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>相关系数阈值(请输入)</string>
     </property>
    </widget>
    <widget class="QLabel" name="LSMepsilon">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>120</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>收敛阈值</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="epsilon">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <double>0.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.001000000000000</double>
     </property>
     <property name="value">
      <double>0.000000000000000</double>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="run">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>571</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>run</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelResult">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>20</y>
      <width>801</width>
      <height>411</height>
     </rect>
    </property>
    <property name="text">
     <string>匹配结果</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignBottom|Qt::AlignmentFlag::AlignHCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="PointExport">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>571</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>同名点导出设置</string>
    </property>
    <widget class="QCheckBox" name="ExportPoints">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>541</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>是否输出同名点</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelExportStyle">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>选择导出格式</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SelectOutPath">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>90</y>
       <width>91</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>SelectOutPath</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ExportPath">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>461</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QRadioButton" name="txtStyle">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>60</y>
       <width>96</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>TXT</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="csvStyle">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>60</y>
       <width>96</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>CSV</string>
     </property>
    </widget>
    <zorder>ExportPoints</zorder>
    <zorder>labelExportStyle</zorder>
    <zorder>SelectOutPath</zorder>
    <zorder>ExportPath</zorder>
    <zorder>csvStyle</zorder>
    <zorder>txtStyle</zorder>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1460</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Match.qrc"/>
 </resources>
 <connections/>
</ui>
